hicommonwealth/commonwealth
# Enable CE 2

## Link to Issue
Closes: #5137 

## Description of Changes
- Removes the create, update, delete (CUD) rascal configurations and consumer/producer functionality for chain-event notifications.
  - This removes the last queue that exists between chain-events and commonwealthapp but does not disable chain-events entirely, that is left to #5247.
- Adds evm-ce to the commonwealthapp Procfile
- Renamed `tsconfig.consumer.json` to `tsconfig.worker.json`

## Test Plan
- Check `evm-ce` dyno logs on Heroku
- Ensure normal functionality of chain subscriptions on the notification settings page.

This is difficult to test because real proposals only come by every so often so it takes time. Devnet tests include proposal creation + fast-forwarding so this test plan is probably unnecessary.
- Join `aave` community.
- Go to notification settings page and subscribe to `aave` chain notifications
- Wait ~10 days for a new proposal to be created and therefore a new notification to be emitted.

## Deployment Plan

1. Deploy to CW
3. Once finished deployed to CE
4. Delete the notifications CUD queue (`ChainEventNotificationsCUDMainQueueV2`) and exchange (`CreateUpdateDeleteExchange`) from the RabbitMQ management panel.
5. Scale the `evm-ce` to a single dyno in CW.

## Other Considerations

-  In draft until #5136 is merged.
